docker-storage-setup.sh

```diff
@@ -1017,6 +1060,13 @@ setup_storage() {
     Error "Failed to setup docker root volume."
     return 1
   fi
+
+  # Set up a separate named ($EXTRA_VOLUME_NAME) volume
+  # for $EXTRA_VOLUME_MOUNT_DIR.
+  if ! setup_extra_lv_fs; then
+    Error "Failed to setup logical volume for $EXTRA_VOLUME_MOUNT_DIR."
```

Comment:
Lets say "Failed ot setup extra volume $EXTRA_VOLUME_NAME.

We are giving similar message inside the function too. So may be get rid of the one inside the function so that there are no duplicate messages.